common/courses/notifications.ts

```diff
@@ -117,6 +117,12 @@ export async function sendPupilCoursePromotion(subcourse: Prisma.subcourse, coun
             active: true,
             verification: null,
             isParticipant: true,
+            pupil_screening: {
+                some: {
+                    invalidated: false,
```

Comment:
why?

Reply:
I assumed that we don't want to send emails to these people either, as they cannot join courses anyways, as their screening is invalidated?

Reply:
they can. Invalidation is only relevant for matching